TestHelper/TestHelper.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>

```

Comment:
Can it not target `net6.0`?

Reply:
Discussed offline, I chose netstardard2.1 out of old habits for creating a Library project. While this has greater backwards compatibility - we do not need backwards compatibility, and this limits the C# language version, so I will update this to `net6.0` so we have one language version across all projects.

Reply:
TestHelper is a library, so it is not necessary to target net6.0